src/components/Dashboard/_dashboard.scss

```diff
@@ -60,3 +60,7 @@
     }
   }
 }
+// workaround to hide the resize handles in react-grid-layout
+.react-resizable-hide .react-resizable-handle {
```

Comment:
Could we scope this under the root dashboard style? It's possible this is used elsewhere outside of dashboard (now or in the future) and I don't think we'd want to globally override this.

Reply:
This might need to go under `DashboardGrid`'s style sheet since this component is being deprecated

Reply:
Ah yeah good call - let's move this there and scope it under the root DashboardGrid class.